File: C++ for Everyone, 2nd Ed, Horstmann/Chapter1/P1-4.cpp
//Write a program that prints the sum of the first ten positive integers, 1 + 2 + … + 10.

#include<iostream>

using namespace std;

int main()
{
    int sum = 0;
    for (int i = 1; i <= 10; i++)
    {
        sum += i;
    }

    cout << sum << "\n";

    return 0;
}

File: Java-for-Everyone-Late-Objects-Horstmann/Chapter-1-Introduction/P1_4.java
/*
Write a program that prints the balance of an account
after the first, second and third year.
Initial balance is $10000 and interest per year - 5%
*/


public class P1_4 {
    public static void main(String[] args) {
        System.out.print("Account balance:");
        int account_balance = 10000;
        System.out.println(account_balance);

        System.out.print("\nInterest per year: ");
        int interest_per_year = 5;
        System.out.println(interest_per_year);

        double balance_after_first_year = account_balance + account_balance * interest_per_year;
        System.out.print("Balance after first year: ");
        System.out.println(balance_after_first_year);

        double balance_after_second_year = balance_after_first_year * 2;
        System.out.print("Balance after second year: ");
        System.out.println(balance_after_second_year);

        double balance_after_third_year = balance_after_first_year * 3;
        System.out.print("Balance after third year: ");
        System.out.println(balance_after_third_year);

    }
}
